### PR TITLE
Haproxy test

### PR DESCRIPTION
**Pull request by razvan-moj**
**Why**
connects to https://github.com/ministryofjustice/cloud-platform/issues/430

### CODE REVIEW

File: haproxy/alb.tf [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    Name = "haproxy_alb"
+  }
+}
+
+resource "aws_lb_target_group" "haproxy_alb_target" {
+  name     = "haproxy-test-alb-tg"
+  vpc_id   = "${aws_vpc.default.id}"
+  protocol = "HTTP"
+  port     = 80
+
```
Comment (jasonBirchall) [inline haproxy/alb.tf]: Ran `terraform plan` and it resulted in the following:
```
Terraform will perform the following actions:

  ~ aws_lb_target_group.haproxy_alb_target
      proxy_protocol_v2: "" => "false"


Plan: 0 to add, 1 to change, 0 to destroy.
```
Perhaps:
```suggestion
  proxy_protocol_v2 = "false"
```

File: haproxy/alb.tf [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+resource "aws_lb_target_group" "haproxy_alb_target" {
+  name     = "haproxy-test-alb-tg"
+  vpc_id   = "${aws_vpc.default.id}"
+  protocol = "HTTP"
+  port     = 80
+# not sure about this one
+# proxy_protocol_v2 = "false"
```
Comment (jasonBirchall) [inline haproxy/alb.tf]: Can you take this out completely?

File: haproxy/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Weighted load balancer using haproxy and ALB
+Edit `variables.tf` and `terraform apply`
+
```
Comment (jasonBirchall) [inline haproxy/README.md]: Can you add something about any pre-req work required i.e. create a hosted zone?

Comment (razvan-moj) [reply haproxy/README.md]: not needed anymore! 😄

**Pull request merged**